app/code/Magento/Catalog/Test/Mftf/Test/StorefrontVerifyProductAfterPartialReindexOnSeveralWebsitesTest.xml

```diff
@@ -83,7 +83,7 @@
 
         <!-- Run cron -->
         <magentoCLI command="cron:run" stepKey="runCron"/>
-
+        <waitForPageLoad time="60" stepKey="waitForCron"/>
```

Comment:
@engcom-Charlie this change looks out of scope for this pull request. Please can you move this to a separate pull request.